cmp/grid/Grid.js

```diff
@@ -232,6 +232,8 @@ class LocalModel {
             defaultGroupSortComparator: this.groupSortComparator,
             groupDefaultExpanded: 1,
             groupUseEntireRow: true,
+            groupRowInnerRenderer: model.groupRowRenderer,
```

Comment:
Yes - we use the inner renderer, so it just changes the string between the arrows and the numbers. If you want to do more complex customisations, you should use the element version.
<img width="360" alt="Screenshot 2020-02-20 at 14 15 45" src="https://user-images.githubusercontent.com/3017757/74941827-9cb8d400-53eb-11ea-9519-ef08d1f2702e.png">

Reply:
If you specify a `groupRowRenderer` like we currently do for DataView, it will override pretty much everything. So no expand/collapse arrows.

If you specify a `groupRowInnerRenderer` like you do here, it will keep the default expand/collapse arrows.